regexes/device/mobiles.yml

```
@@ -42655,6 +42695,17 @@ DUDU AUTO:
   device: 'car browser'
   model: 'DUDU$1'
 
+# Temigereev
+Temigereev:
+  regex: 'TG ?(15|30)(?:[);/ ]|$)'
+  device: 'tablet'
+  model: 'TG $1'
+
+# Blackton (https://blackton.tv)
+Blackton:
+  regex: 'Bt[_ -](\d{2}FS\d{2}B)(?:[);/ ]|$)'
+  device: 'tv'
+  model: 'Bt $1'
```

Since `Bt` comes from `Blackton`, probably is best to leave this out for model?

```suggestion
  model: '$1'
```

---

https://blackton.tv/catalog/tv/bt-32fs32b/